### ISSUE TITLE
"jsonnet_to_json" "imports" attribute does not accommodate empty package name

### ISSUE THREAD

**Issue by seh** [bazelbuild/rules_jsonnet#55]
If you try to use the "imports" attribute of the `jsonnet_to_json` rule in a package without a name, the `-J` parameter formed for each entry winds up mistakenly specifying an absolute path starting at the root directory, instead of a relative path as intended.

Consider a project that has the following structure:
```
top
├── BUILD
├── WORKSPACE
├── service.jsonnet
└── lib
    ├── BUILD
    └── stuff.libsonnet
```
If in the _top/BUILD_ file we write the following, Bazel will compose the `-J` argument as "/lib," instead of "lib" or "./lib":
```
load("@io_bazel_rules_jsonnet//jsonnet:jsonnet.bzl", "jsonnet_to_json")

jsonnet_to_json(
    name = "service",
    src  = "service.jsonnet",
    deps = ["//lib:stuff"],
    imports = ["lib"],
    outs = [
        "svc-1",
        "svc-2",
    ],
)
```
The problem is in function `_jsonnet_to_json_impl`, where [it expects that `ctx.label.package` will be nonempty](https://github.com/bazelbuild/rules_jsonnet/blob/master/jsonnet/jsonnet.bzl#L113):
```python
["-J %s/%s" % (ctx.label.package, im) for im in ctx.attr.imports] +
```
In this case, since we're at the root of the workspace, that attribute is empty. Instead, that should be something more like this:
```python
["-J %s%s" % (ctx.label.package + '/' if len(ctx.label.package) > 0 else "", im) for im in ctx.attr.imports] +
```
That is, only include the separating slash if there's a preceding package path to use as the parent path for the import path.

While I'm here, I'll note that it would be nice to not need to specify that "imports" attribute, as it looks like it could be gleaned from the paths to the "deps" entries.

**Comment by seh** [comment]
This behavior came in with commit 0556c14f709a3e81604ccec30e6048894eb772b7 by @davidzchen.

**Comment by seh** [comment]
It looks like @mariusgrigoriu's #46 would fix this problem with commit 12222d209ad96fa1325dc6b6034280040c68f354.

**Comment by mariusgrigoriu** [comment]
I'm not too happy with that PR. Let me give it another shot.
